apps/graphql/src/apps/itinerary/dataloaders/Itineraries.js

```diff
@@ -51,6 +52,29 @@ export const parseParameters = (input: ItinerariesSearchParameters) => {
   return params;
 };
 
+export const parseParametersNew = (
+  input: ItinerariesOneWaySearchParameters, // Later on will be expanded to be the return one
```

Comment:
What about mark it with `@TODO` in the comment ?

Reply:
Done